lib/govspeak.rb

```diff
@@ -406,6 +406,20 @@ def self.devolved_options
       %(<iframe class="govspeak-embed-video" width="500" height="281" src="#{embed_url}" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe>)
     end
 
+    extension("Figure", /#{NEW_PARAGRAPH_LOOKBEHIND}\$Figure\s*$(.*?)\s*\$EndFigure/m) do |figure|
```

Comment:
I don't think we want the `#{NEW_PARAGRAPH_LOOKBEHIND}` part. That was what was breaking accordions on some pages.